src/features/retreat/components/blacksmith/components/GoblinBlacksmithItems.tsx

```diff
@@ -74,6 +74,8 @@ export const GoblinBlacksmithItems: React.FC<Props> = ({ onClose }) => {
       selectedItem.ingredients[name]?.greaterThan(inventory[name] || 0)
     );
 
+  const lessSFL = () => selectedItem.sfl && selectedItem.sfl.gt(state.balance);
```

Comment:
The "less" naming is confusing (and weird to me, maybe it's a region thing).  Recommend "missing".